coordinator/src/services/translation/index.ts

```diff
@@ -100,3 +101,9 @@ class AlgorithmIdentifier implements XmlJs.ElementCompact {
     }
   }
 }
+
+export function convertFhirMessageToReleaseRequest(fhirMessage: fhir.Parameters): SpineRequest {
```

Comment:
would it make more sense to call this `convertParametersToSpineRequest`? just comparing to the function above (convertFhirMessageToSpineRequest), seems like the difference between these two functions is what goes in (not what comes out). so potentially change that to `convertBundleToSpineRequest` too?